src/Nonunit/Review/SegmentBloodVesselsWithMultiScaleHessianBasedMeasure/Code.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# Copyright Insight Software Consortium
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+import itk
+from distutils.version import StrictVersion as VS
+if VS(itk.Version.GetITKVersion()) < VS("5.0.0"):
+    print("ITK 5.0.0 or newer is required.")
+    sys.exit(1)
+
+parser = argparse.ArgumentParser(description='Segment blood vessels with multi-scale Hessian-based measure.')
+parser.add_argument('input_image')
+parser.add_argument('output_image')
+parser.add_argument('--sigma_minimum', type=float, default=1.0)
+parser.add_argument('--sigma_maximum', type=float, default=10.0)
+parser.add_argument('--number_of_sigma_steps', type=int, default=10)
+args = parser.parse_args()
+
+input_image = itk.imread(args.input_image, itk.ctype('float'))
+
+objectness_filter = itk.hessian_to_objectness_measure_image_filter[itk.ctype('float')].New()
+objectness_filter.SetBrightObject(false);
+objectness_filter.SetScaleObjectnessMeasure(false);
+objectness_filter.SetAlpha(0.5);
+objectness_filter.SetBeta(1.0);
+objectness_filter.SetGamma(5.0);
+
+multi_scale_enhancement_filter = itk.multiscale_hessian_based_measure_image_filter(input_image)
```

Comment:
Here we want to use the procedural or the object oriented interface.

For the object oriented, a `New()` is missing, i.e.

```
multi_scale_enhancement_filter = itk.multiscale_hessian_based_measure_image_filter.New(input_image)
```

For the procedural interface:

```
enhanced = itk.multiscale_hessian_based_measure_image_filter(input_image,
  hessian_to_measure_filter=objectness_filter,
  sigma_minimum=args.sigma_minimum,
  sigma_maximum=args.sigma_maximum,
  number_of_sigma_steps=args.number_of_sigma_steps)
```

Since a procedural interface is what Python folks are more used to, we should use it when possible.

Reply:
Thanks for the review @thewtex . Done in d4a071d.